### ISSUE TITLE
Countries - Legal Standards - VoiceOver Reading N/A Incorrectly

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#86]
VoiceOver is reading "N/A" as such, but on other screens, it is read as "Not available".

Examples of this issue can be found in the following countries:
- Brazil
- India
- Jamaica
- Uganda

**Comment by jfc3-dol** [comment]
When reading the page with VoiceOver it's reading the "**N/A**" items as "**Not Available**". But the "**No**" item as "**Not Available**" instead of "**No**". At least this is happening for **India**,  and **Uganda**. It seems to be for a "**No**" item that follows an "N/A" item.

Also in **India**, **Venezuela**, and **Turkey** VoiceOver reads the "State Compulsory Military Recruitment" "Yes" as "Yes Not Available".

Not sure if this one got set that way by mistake. There could be more.

**Comment by jfc3-dol** [comment]
I'm still seeing the issues mentioned above.

**Comment by sanganalakshay** [comment]
should be fixed in the latest commit

**Comment by jfc3-dol** [comment]
Reviewed the list of countries above along with about ten more and VoiceOver read the answers/responses on the "Legal Standards" screens as expected.

Closing issue.
